src/Utility/TextProcessingTrait.php

```diff
@@ -75,6 +76,12 @@ public function createSummary($text, $size = 300)
             }
         }
 
+        // If ellipsis is set to true, add trailing points at the end of the string
+        if ($ellipsis) {
+            $ending = ' ...';
+            $summary = $summary . $ending;
```

Comment:
This could be replaced with:
```
if ($ellipsis) {
    return $summary . ' ...';
}
```

Reply:
Yes I figured it would be clearer like this